### ISSUE TITLE
Proxy code training results

### ISSUE THREAD

**Issue by MikeWangWZHL** [bytedance/1d-tokenizer#18]
Hi, is it possible to share the reconstruction results / visuals from only stage 1 proxy code training? (with cross-entropy loss) It would be very helpful to see how much we can get with only cross-entropy on the VQGAN codes;
Thanks!

**Comment by cornettoyu** [comment]
Hi,

Thank you all for your interests in our work and patience. We have released the two-stage training code and feel free to check it. Hope this help addresses your question.
